docs/standards.md

```diff
@@ -0,0 +1,52 @@
+# Mapping Standards
+
+This document details the standards that all contributions to Parchment mappings must follow.
+
+### Parameter Names
+
+1. Use American English for spellings
+    - Example: `color`, not `colour`, `armor`, not `armour`
+2. All names should be prefixed with `p`
+    - Example: `pLevel`, `pXPos`
+3. Names should make sense without the prefix, and should be valid java identifiers NOT reserved keywords as such:
+    - Example: `pFloat` is NOT valid, as `float` is a reserved keyword.
+    - Example: `pOs` is not a correct use of the prefix, it should be `pPos`
+4. Parameter names may be named based on the types, and the context in which they are used. They should be verbose and use complete words - do not omit essential information for brevity's sake.
+    - Example: `BlockPos pAdjacentPos, BlockPos pCurrentPos`, not `BlockPos pPos1, BlockPos pPos2`
+5. Avoid single letters, or abbreviations.
+    - Example: `pNorthWest` over `pNW`, `pMatrixStack` over `pMStack`. 
+    - Exception: common abbreviations can be used: `IO` / `NBT`, etc.
+6. Use Lower Camel Case (`lowerCamelCase`)
+7. Do not use `$` or `_` in variable names
+8. All parameters should match the regex `p[A-Z][A-Za-z0-9]+`.
+    - Example: `pDot`, `pPos`, `pBlockState`
+9. Favor using Mojang / Mojmap class names over <1.17 MCP classes. In general, the mojmap name is the de-facto standard (some exceptions may apply).
+    - Example: Use `level` over `world`, `blockEntity` over `tileEntity`
+10. Common parameter names can use abbreviations for brevity:
+    - Example: `BlockPos pos, Level level, BlockState state, WorldGenLevel level`
```

Comment:
Just realized this parameter list itself is invalid if within a method signature. Want to avoid any ambiguity that might come or would result in a later PR of people wondering the same.